>From 5c9f263416f9842d4f901f53cad19386178141dc Mon Sep 17 00:00:00 2001
From: Daniele Forsi <dforsi@gmail.com>
Date: Sun, 19 May 2013 18:45:29 +0200
Subject: [PATCH] Fix syntax error in CSS file

Fixes the following errors printed by build/build-epub:
PropertyValue: Unknown syntax or no value:  0px
	text-align: center
CSSStyleDeclaration: Syntax Error in Property: border-bottom: 0px
	text-align: center
---
 build/epub/tdah.css |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/epub/tdah.css b/build/epub/tdah.css
index 210d6f2..e5e9d65 100644
--- a/build/epub/tdah.css
+++ b/build/epub/tdah.css
@@ -64,7 +64,7 @@ div.url + div.url {
 	padding: 5px;
     }
     div.sidebar-title {
-	border-bottom: 0px
+	border-bottom: 0px;
 	text-align: center;
 	text-indent: 0;
     }
-- 
1.7.10.4

